editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3817,12 +3818,16 @@ void CanvasItemEditor::_draw_viewport() {
 		}
 	}
 
+	force_disable_lock_group = selection.size() == 1 && Object::cast_to<CanvasItem>(selection[0]) == nullptr;
```

Comment:
The button will be always enabled when you select more nodes. You should check whether there is any CanvasItem in the selection.

Reply:
> when you select more nodes

Sorry I don't quite understand your suggestion. In this case `force_disable_lock_group` will be false and all buttons will act like before (before this pr is introduced).

Reply:
Yep, which means that you can select e.g. 2 plain Nodes and the button will be enabled and do nothing. So right now the PR only fixes a case for single selected node.

Reply:
I see, thanks!